app/src/components/market/sections/market_profile/market_status/open.tsx

```diff
@@ -263,8 +275,18 @@ const Wrapper = (props: Props) => {
         ></MarketNavigation>
         {currentTab === MarketDetailsTab.swap && (
           <>
-            {renderTableData()}
-            {isQuestionOpen && openQuestionMessage}
+            {isScalar ? (
+              <MarketScale
+                border={true}
+                currentPrediction={outcomeTokenMarginalPrices ? outcomeTokenMarginalPrices[1] : null}
+                lowerBound={scalarLow || new BigNumber(0)}
+                startingPointTitle={'Current prediction'}
+                unit={question.title ? question.title.split('[')[1].split(']')[0] : ''}
```

Comment:
This should be refactored to take the last occurance of [...] to prevent bugs where [] is part of the string
Check this example
![image](https://user-images.githubusercontent.com/4677914/103885837-34fa9d00-5113-11eb-944e-dd76a75b22df.png)